src/components/Navbar/NavDropdown.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { useQuery } from "@apollo/react-hooks";
+import styled from "@emotion/styled";
+import { Menu, Dropdown } from "antd";
+import Icon from "@leafygreen-ui/icon";
+import { uiColors } from "@leafygreen-ui/palette";
+import get from "lodash/get";
+import { Link } from "react-router-dom";
+
+import { GET_USER } from "gql/queries";
+import { GetUserQuery } from "gql/generated/types";
+import { getUiUrl } from "utils/getEnvironmentVariables";
+import { legacyRoutes } from "constants/externalResources";
+import { paths, preferencesTabRoutes } from "constants/routes";
+
+const { white } = uiColors;
+
+export const NavDropdown = () => {
+  const { data } = useQuery<GetUserQuery>(GET_USER);
+  const displayName = get(data, "user.displayName");
+  return (
+    <Dropdown overlay={MenuItems}>
+      <NavDropdownTitle
+        className="ant-dropdown-link"
+        onClick={(e) => e.preventDefault()}
+      >
+        {displayName}
+        <Icon glyph="CaretDown" />
+      </NavDropdownTitle>
+    </Dropdown>
+  );
+};
+
+const MenuItems = () => {
+  const uiURL = getUiUrl();
+  return (
+    <Menu>
+      <Menu.Item>
```

Comment:
Are the contents here meant to allow navigation similar to the old UI without having to flip back and forth? There are a couple things missing that can be added later, but I think it would make sense to add a link to the user patches page here in this PR

Reply:
This is meant to provide links to the legacy ui for pages that arent yet available in spruce.

Reply:
From the slack thread, I think we should add a link to the my patches page as well. That's where the current patches page lives and is where users will be looking for it